resources/packs/gcp/dataproc.go

```diff
@@ -132,453 +165,453 @@ func (g *mqlGcpProjectDataprocService) GetClusters() ([]interface{}, error) {
 			clusters, err := dataprocSvc.Projects.Regions.Clusters.List(projectId, regionName).Do()
 			if err != nil {
 				log.Error().Err(err).Send()
-			}
-
-			for _, c := range clusters.Clusters {
-				var mqlConfig resources.ResourceType
-				if c.Config != nil {
-					var mqlAutoscalingCfg map[string]interface{}
-					if c.Config.AutoscalingConfig != nil {
-						type mqlAutoscalingConfig struct {
-							PolicyUri string `json:"policyUri"`
-						}
-						mqlAutoscalingCfg, err = core.JsonToDict(mqlAutoscalingConfig{PolicyUri: c.Config.AutoscalingConfig.PolicyUri})
-						if err != nil {
-							log.Error().Err(err).Send()
+			} else {
```

Comment:
If there's an err it's likely that the `for` down below will not work, hence the `else` clause

We can improve this code for readability